recipes/phps-mode

```diff
@@ -0,0 +1 @@
+(phps-mode :repo "cjohansson/emacs-phps-mode" :fetcher github)
```

Comment:
The default recipe will:
```elisp
(:exclude ".dir-locals.el" "test.el" "tests.el" "*-test.el" "*-tests.el"))
```
So it _will_ exclude `phps-mode-test.el`, but it won't exclude:
- phps-mode-test-lexer.el
- phps-mode-test-functions.el
- phps-mode-test-syntax-table.el
- phps-mode-test-parser.el
- phps-mode-test-integration.el
So if these are test files, you should modify this recipe.

Reply:
What does test files mean? Those files are used for unit and integration testing using GNU make and Travis

Reply:
Oh yeah, that's what I meant.  :)  Sorry for lack of clarity.  With this melpa recipe and others, you don't want to distribute the elisp files you use for testing to users.  So your recipe should exclude them, or you can move the test files into a subdirectory called "test" or similar.

Reply:
Ok I understand :) I'll modify the recipe then